modules/apps/portal-workflow/portal-workflow-task-web/src/main/resources/META-INF/resources/css/main.scss

```diff
@@ -309,6 +309,14 @@
 	.metadata-author {
 		font-weight: bold;
 		margin-right: 10px;
+
+		img {
+			vertical-align: baseline;
+		}
+
+		svg {
+			vertical-align: baseline;
+		}
```

Comment:
@naraandrade you could use `img, svg { ... }` to apply the same style to one more selector.

Reply:
> @naraandrade you could use `img, svg { ... }` to apply the same style to one more selector.

Thanks, @albinojunior :) I'll do that